### PR TITLE
removed broken link added new correct link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #8473 , so i replaced the broken link to correct link.

### CODE REVIEW

File: windows/security/threat-protection/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ ms.topic: conceptual
 ---
 
 # Threat Protection
-[Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559) is a unified platform for preventative protection, post-breach detection, automated investigation, and response. Microsoft Defender ATP protects endpoints from cyber threats; detects advanced attacks and data breaches, automates security incidents and improves security posture.
+[Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP)](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection) is a unified platform for preventative protection, post-breach detection, automated investigation, and response. Microsoft Defender ATP protects endpoints from cyber threats; detects advanced attacks and data breaches, automates security incidents and improves security posture.
 
 >[!TIP]
```
Comment (illfated) [inline windows/security/threat-protection/index.md:22]: Link above is fine. Could we also allow for a minor addition, a codestyle improvement here at the same time?
```suggestion
> [!TIP]
```

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — COMMENTED** [review]
Looks good.

**Pull request merged**